### PR TITLE
Minor updates to Schema Registry SDK for private preview

### PR DESCRIPTION
**Pull request by arerlend**
@srnagar 

Just some minor changes made after doing some manual validation against actual service.

We've changed uuid to String for ID format, so client was re-generated.  Swagger in client directory is up to date.

### CODE REVIEW

File: sdk/schemaregistry/azure-data-schemaregistry/src/main/java/com/azure/data/schemaregistry/client/CachedSchemaRegistryClientBuilder.java [hunk 1]
```diff
@@ -74,6 +76,10 @@ public CachedSchemaRegistryClientBuilder() {
         this.credential = null;
         this.retryPolicy = new RetryPolicy("retry-after-ms", ChronoUnit.MILLIS);
 
+        HttpHeaders headers = new HttpHeaders();
+        headers.put("api-version", "2017-04");
```
Comment (srnagar) [inline sdk/schemaregistry/azure-data-schemaregistry/src/main/java/com/azure/data/schemaregistry/client/CachedSchemaRegistryClientBuilder.java:80]: The version usually is the year and month when the service is released. Why is this set to 2017? Also, to set the service version, use `ServiceVersion` interface and define an enum like this - [ConfigurationClientBuilder](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/appconfiguration/azure-data-appconfiguration/src/main/java/com/azure/data/appconfiguration/ConfigurationClientBuilder.java#L381)

Comment (arerlend) [reply sdk/schemaregistry/azure-data-schemaregistry/src/main/java/com/azure/data/schemaregistry/client/CachedSchemaRegistryClientBuilder.java:80]: We will make this configurable in the next release.

Comment (arerlend) [reply sdk/schemaregistry/azure-data-schemaregistry/src/main/java/com/azure/data/schemaregistry/client/CachedSchemaRegistryClientBuilder.java:80]: Adding todo in my local branch.

**Pull request merged**